src/runtime/getGlobalThis.js

```diff
@@ -4,28 +4,28 @@ module.exports = (function () {
     return globalThis;
   }
 
-  let g;
+  let globalObj;
 
   try {
     // This works if eval is allowed (see CSP)
     // eslint-disable-next-line no-new-func
-    g = this || new Function("return this")();
-  } catch (e) {
+    globalObj = this || new Function("return this")();
+  } catch {
     // This works if the window reference is available
-    if (typeof window === "object") {
-      return window;
+    if (typeof globalThis.window === "object") {
+      return globalThis;
```

Comment:
In this place lets keep window for tests, just use `// eslint-disable-next-line`